src/templates/items/delete-container.html

```diff
@@ -0,0 +1,6 @@
+<form class="ose dialog">
+    <div>
+        {{localize 'OSE.dialog.confirmDeleteContainer'}}
+        Are you sure you want to delete the container?
```

Comment:
Should this be localized?

Reply:
You're right! I missed to remove my non-localized text!